### ISSUE TITLE
Outdated --kubernetes-version references

### ISSUE THREAD

**Issue by philip** [vitessio/website#1022]
The latest [Getting Started: Operator](https://vitess.io/docs/14.0/get-started/operator/) suggests installing an oldish and specific version of Kubernetes:

```
minikube start --kubernetes-version=v1.19.16 --cpus=4 --memory=4000 --disk-size=32g
```

References:

```
content/en/docs/14.0/get-started/operator.md
  minikube start --kubernetes-version=v1.19.16 --cpus=4 --memory=4000 --disk-size=32g
content/en/docs/13.0/get-started/operator.md
  minikube start --kubernetes-version=v1.19.16 --cpus=4 --memory=4000 --disk-size=32g
content/en/docs/12.0/get-started/operator.md
  minikube start --kubernetes-version=v1.19.16 --cpus=4 --memory=4000 --disk-size=32g
content/en/docs/11.0/get-started/operator.md
  We recommend using Kubernetes 1.14, as this is a common denominator across public clouds:
  minikube start --kubernetes-version=v1.14.10 --cpus=8 --memory=11000 --disk-size=50g
```

There's also a similar (but older) reference in `vitess.git`'s example operator readme:

```
examples/operator/README.md
  minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.19.5
```
Suggestion is to either:

  A. Remove the `--kubernetes-version` parameter and instead rely upon the [prerequisites documentation](https://github.com/planetscale/vitess-operator#compatibility) it links to.

  B. Or only add `--kubernetes-version` to older versioned docs if the latest stable isn't supported, and if it won't change.

Option (B) risks becoming outdated (e.g., `examples/operator/README.md`) although `--kubernetes-version=v1.19.16` may be appropriate for `docs/11.0/` and `docs/12.0/` today, although the prerequisites matrix is missing v11.0 so I'm not sure.
 
FWIW I prefer (A) to avoid future doc maintenance and bug reports like this. Regardless, I can create a PR if folks agree upon a solution as I'm not familiar enough with Vitess to know for certain.

Related: I hoped `--kubernetes-version=` didn't require a patch version but [it _appears_ required](https://minikube.sigs.k8s.io/docs/handbook/config/#selecting-a-kubernetes-version), else the likes of `--kubernetes-version=v1.19` would be possible with (B).

**Comment by deepthi** [comment]
@GuptaManan100 @frouioui @verolop what would you recommend?

**Comment by frouioui** [comment]
I like option A as it has a single point of maintenance. We would only need to update the [compatibility table](https://github.com/planetscale/vitess-operator#compatibility) of the vitess operator. However, as a user, I think it is nice to just be able to copy/paste command lines from the getting started guide.

I think we should remove the instructions in `examples/operator/README.md` and instead link to the documentation. That way we can have all the documentation regarding the getting started guide in a single place.

I lean more towards an option between A and B where we would continue updating the vitess operator compatibility table, and start updating the getting started guide with the proper kubernetes versions. But I do not have strong opinions.

**Comment by anshikavashistha** [comment]
@frouioui @deepthi I have gone through the release notes of Vitess-operator but want to discuss regarding the modifications on   [Compatiblity Matrix](https://github.com/planetscale/vitess-operator) .
Thank you

**Comment by deepthi** [comment]
I think we can close this given vitessio/vitess#16610

**Comment by philip** [comment]
It doesn't appear to me that https://github.com/vitessio/vitess/pull/16610 fixes this (bug #1022) issue, as this references Getting Started along with README.md. For example, this file (and for the other versions, not just 20.0):

https://github.com/vitessio/website/blob/prod/content/en/docs/20.0/get-started/operator.md ->
https://vitess.io/docs/20.0/get-started/operator/

The example there today is:

`minikube start --kubernetes-version=v1.28.5 --cpus=4 --memory=11000 --disk-size=32g`
